### PR TITLE
allow setting of kibana env vars

### PR DESCRIPTION
**Pull request by jcantrill**
This PR allows setting kibana env vars via ansible in support of being able to adjust kibana config via env vars

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @pweil-

**Comment by jcantrill** [comment]
@bmcelvee when this pr merges you will be able to configure kibana using env vars: https://github.com/openshift/origin-aggregated-logging/pull/905/files#diff-96df006f083f3cf4a36be860db024795R8  
Could you add a docs pr to support?

File: roles/openshift_logging/README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ When `openshift_logging_install_logging` is set to `False` the `openshift_loggin
 - `openshift_logging_kibana_replica_count`: The number of replicas Kibana should be scaled up to. Defaults to 1.
 - `openshift_logging_kibana_nodeselector`: A map of labels (e.g. {"node":"infra","region":"west"} to select the nodes where the pod will land.
 - `openshift_logging_kibana_edge_term_policy`: Insecure Edge Termination Policy. Defaults to Redirect.
+- `openshift_logging_kibana_env_vars`: A map of environment variables to add to the kibana deployment config (e.g. {"ELASTICSEARCH_REQUESTTIMEOUT":"30000"})
```
Comment (ewolinetz) [inline roles/openshift_logging/README.md]: nit, but can we make sure if that time should be quoted or not for the example? (Is it evaluated as a string or an int)

Comment (ewolinetz) [inline roles/openshift_logging/README.md]: Do we want to give the ability to configure `ops` and non-ops differently?

**Comment by bmcelvee** [comment]
@jcantrill sure thing!

**Comment by ewolinetz** [comment]
bot, retest this please

**Pull request merged**